src/Symfony/Component/HttpFoundation/BinaryFileResponse.php

```diff
@@ -350,7 +350,7 @@ public static function trustXSendfileTypeHeader()
      *
      * @return BinaryFileResponse
      */
-    public function deleteFileAfterSend($shouldDelete)
+    public function deleteFileAfterSend($shouldDelete = true)
```

Comment:
@chalasr I'll improve the docs, and I'll redo this pull request (including a new `keepFileAfterSend()`) later.

Thanks for review :)